.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,56 @@
+name: Build
+
+# Only trigger, when the build workflow succeeded
+on:
+  workflow_run:
+    workflows: ["CI build"]
```

Comment:
Ops, this should have been
Publish to PyPI
`    workflows: ["Publish to PyPI"]`

Reply:
Gotcha, let me fix that.